cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CombiningOperatorsTest.scala

```diff
@@ -131,11 +131,11 @@ class CombiningOperatorsTest extends DocumentingTest {
       }
     }
     section("AntiSemiApply", "query-plan-anti-semi-apply") {
-      p("""Tests for the existence of a pattern predicate.
-           |`SemiApply` takes a row from its child operator and feeds it to the leaf operator on the right-hand side.
+      p("""Tests for the absence of a pattern predicate.
+           |`AntiSemiApply` takes a row from its child operator and feeds it to the leaf operator on the right-hand side.
            |If the right-hand side operator tree yields at least one row, the row from the
-           |left-hand side is yielded by the `SemiApply` operator.
-           |This makes `SemiApply` a filtering operator, used mostly for pattern predicates in queries.""")
+           |left-hand side is yielded by the `AntiSemiApply` operator.
+           |This makes `AntiSemiApply` a filtering operator, used mostly for pattern predicates in queries.""")
```

Comment:
Here I think the word "predicate" is correctly used, although we can leave out the "mostly" - it's only used for that.